### PR TITLE
Story1 show collection

### PR DESCRIPTION
**Pull request by PNaing107**
Please review my code for Story 1.

### CODE REVIEW

File: CreateInitialTables.sql [hunk 1]
```diff
@@ -0,0 +1,244 @@
+-- 1. Create a Database called epic-rides
+-- 2. Copy and paste the following SQL commands into Sequel Ace and run them to setup the Tables
+DROP TABLE IF EXISTS `locations`;
+
+CREATE TABLE `locations` (
+                             `location_id` int(11) unsigned NOT NULL AUTO_INCREMENT,
```
Comment (nealioa) [inline CreateInitialTables.sql]: The indentation here is a bit much. Just 4 spaces would be good.

File: CreateInitialTables.sql [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+        'https://h5d9a9f8.rocketcdn.me/wp-content/uploads/2017/10/Mountainbike-Tour-Norway-Photo-Blog-27.jpg',
+        'Mefjellet'
+        );
+INSERT INTO `images` (`image_id`, `route_id`, `url`, `alt_text`)
+VALUES (12,
+        12,
+        'https://www.moredirt.com/trails/featured/bike-republic-solden.jpg',
+        'Galitzenklamm Gorge'
+        );
+
```
Comment (nealioa) [inline CreateInitialTables.sql]: Just one blank line at the end of a file is nice.

File: functions.php [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+    return $query->fetchAll();
+}
+
+
+/**
+ * Function to return a string representing HTML for each collection item card
+ * @param array $array
+ * @return string
+ */
+function displayCollection(array $array)
```
Comment (nealioa) [inline functions.php]: Type hint the return.

File: functions.php [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+{
+    //Check if array is empty and show error message if true
+    if(!$array) {
```
Comment (nealioa) [inline functions.php]: Space between `if` and `(`.

File: tests/Functions.php [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+                    'distance' => 123,
+                    'effort_level' => 6,
+                    'adrenaline_rating' =>6
+
+                ]
+            ];
+
+        $expected =
+            '<div class="card">'
+            .'<div class="card-header">'
```
Comment (nealioa) [inline tests/Functions.php]: Space between concatenation operator `.` and the next thing (in this case quote `'`).

File: tests/Functions.php [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+        $input = [];
+        $expected = 'Error: No items have been found please check Database.';
+
+        // Act
+        $result = displayCollection($input);
+
+        // Assert
+        $this->assertEquals($expected, $result);
+
+    }
```
Comment (nealioa) [inline tests/Functions.php:69]: Add a blank line to separate the functions a little.

**Review by nealioa — CHANGES_REQUESTED** [review]

File: CreateInitialTables.sql (hunk 1 above)
Comment (PNaing107) [reply CreateInitialTables.sql]: I have now removed the excess indents.

File: CreateInitialTables.sql (hunk 2 above)
Comment (PNaing107) [reply CreateInitialTables.sql]: Done

File: functions.php (hunk 3 above)
Comment (PNaing107) [reply functions.php]: Added type hint return of string to this function

File: functions.php (hunk 4 above)
Comment (PNaing107) [reply functions.php]: done

File: tests/Functions.php (hunk 5 above)
Comment (PNaing107) [reply tests/Functions.php]: done

File: tests/Functions.php (hunk 6 above)
Comment (PNaing107) [reply tests/Functions.php:69]: done

**Review by nealioa — APPROVED** [review]

**Pull request merged**